okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -270,6 +274,29 @@ SSLSocketFactory createSocketFactory() {
     }
   }
 
+  private static SSLContext getSslContext() throws NoSuchAlgorithmException {
+    if (GrpcUtil.IS_RESTRICTED_APPENGINE) {
+      try {
+        SSLContext tlsContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
+        KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+        keyStore.load(null, null);
```

Comment:
@ejona86 I'd have to dig a bit to see what I did with that code

@lesv we're pre-1.0 ATM.  We have some preliminary snapshots up on the sonatype repo for linux and macos, but nothing official yet.